### PR TITLE
create draft of readme

### PR DESCRIPTION
**Pull request by caioeverest**
## Context

This projects needs a read-me to help users to better understand how the library works and how to use to better take advantage of it's features 

## Changes

Creates Read me file
Add MIT License

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (marcopollivier) [inline LICENSE:1]: Awesome 👏

File: README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    some_key: "test"
+    another_key: 1234
+    sub_object:
+      something: "a"
+      something_else: 1234
+      something_else_with_interpolation: "{SOMETHING_ELSE_WITH_INTERPOLATION:3432.8}"
+    last_key: "{LAST_KEY:default value}"
+    last_key_with_number: "{LAST_KEY_WITH_NUMBER:54}"
+
+As you can see the if you set the value as `"{ENVIRONMENT_VARIABLE:default_value}"` or only `"{ENVIRONMENT_VARIABLE}"`  if you want it to panic if no value no value found for this environment variable is set.
```
Comment (marcopollivier) [inline README.md]:
```suggestion
As you can see the if you set the value as `"{ENVIRONMENT_VARIABLE:default_value}"` or only `"{ENVIRONMENT_VARIABLE}"`  if you want it to panic if no value found for this environment variable is set.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	}
+	
+	type SubObjectType struct {
+		SomeThing                      string
+		SomethingElse                  int
+		SomethingElseWithInterpolation float64
+	}
+
+
+> The "cfg" tag, like the json tag, acts like an alias when binding the keys in the file with your struct's parameters. They are helpful but not obligatory.
```
Comment (marcopollivier) [inline README.md]:
```suggestion
> The "cfg" tag, like the json tag, acts as an alias when binding the keys in the file with your struct's parameters. They are helpful but not obligatory.
```

Comment (marcopollivier) [reply README.md]: I think `as` works better than `like`. WDYT?

**Review by marcopollivier — APPROVED** [review]
LGTM. Just some comments. :)

File: README.md (hunk 3 above)
Comment (caioeverest) [reply README.md]: I agree, is more objective

**Pull request merged**